tools/schemacode/bidsschematools/tests/test_make_testdata.py

```diff
@@ -32,10 +33,10 @@ def test_make_archive(bids_examples, bids_error_examples):
     * Archives will be generated under `/tmp/bidsschematools-testdata-SCHEMA_VERSION.tar.gz`
     """
 
-    target_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), "data", "bundled"))
-    os.makedirs(target_dir, exist_ok=True)
-    shutil.copytree(bids_examples, f"{target_dir}/bids-examples")
-    shutil.copytree(bids_error_examples, f"{target_dir}/bids-error-examples")
+    testdata_dir = files("bidsschematools.tests.data")
+    ignore_git = shutil.ignore_patterns(".git*")
```

Comment:
Got rid of the "bundled" subdirectory, and ignored the `.git*` files.

Reply:
nice :)